pkg/backup/util/util.go

```diff
@@ -410,3 +410,13 @@ func ValidateRestore(restore *v1alpha1.Restore) error {
 	}
 	return nil
 }
+
+// GetImageTag get the image tag
+func GetImageTag(image string) string {
```

Comment:
update it to return `image` and `tag`, e.g. `func PaseImageTag(image string) (string, string)`.
in your case, discard the first result, e.g. `_, tikvVersion := PaseImageTag(...)`.
make it generic, then we can use it in more places.

Reply:
Done.